testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):

```

Comment:
@blmussati I think this sounds like a very good idea! Do you think putting the check in the `__init__` function will be a good idea? Or should we put it in the method `rearrange_all_cars()`?